### ISSUE TITLE
🐛 [BUG]: default handle styles don't correctly adjust handle positions to center on borders if handle size is changed

### ISSUE THREAD

**Issue by snoozbuster** [bcakmakoglu/vue-flow#690]
### Is there an existing issue for this?

- [X] I have searched the existing issues and this is a new bug.

### Current Behavior

Handles are only centered on the edges of the nodes if the handles size are not changed

### Expected Behavior

Handles are centered on the edges of the nodes regardless of the size of the handle

### Steps To Reproduce

set up VueFlow with the additional style

```css
.vue-flow__handle {
  height: 10px;
  width: 10px;
}
```

Observe that handles are no longer centered on the edges of the nodes.

### Anything else?

the [handle styles](https://github.com/bcakmakoglu/vue-flow/blame/3569ebb55c5875ae9780858f5ae99c678e626ef0/packages/core/src/style.css#L183-L206) are all hard-coded to adjust handles by -4px in the opposite direction, instead of using `translate`. For example, code like this works (to my knowledge) for all sizes of handle:

```scss
.vue-flow__handle {
  /* set some width and height... */

  &-bottom {
    bottom: 0;
    transform: translate(-50%, 50%);
  }

  &-top {
    top: 0;
    transform: translate(-50%, -50%);
  }

  &-left {
    left: 0;
    transform: translate(-50%, -50%);
  }

  &-right {
    right: 0;
    transform: translate(50%, -50%);
  }
}
```

I would have just opened this PR myself but I'm not familiar enough with the various styles and configuration options to be sure that this doesn't break something I haven't discovered.

**Comment by bcakmakoglu** [comment]
Thanks for the report, I'll see what can be done :)

**Comment by bcakmakoglu** [comment]
As this kinda constitutes a breaking change, I will hold off releasing the PR for this until the release of `2.0.0`, which might happen rather soon than later. Until then you'll have to overwrite the styles.

**Comment by bcakmakoglu** [comment]
@snoozbuster 
FYI: Will do a pre-release/beta soon for `2.0.0`. Should be by the end of this month but can't make any promises 😄

**Comment by snoozbuster** [comment]
hey @bcakmakoglu. Long time no chat. I am wondering if waiting for a major version release for this is really necessary given this [more significant](https://github.com/bcakmakoglu/vue-flow/issues/1247) breaking change introduced in a minor version, and this [quite significant](https://github.com/bcakmakoglu/vue-flow/issues/1248) breaking change introduced in a patch version. By comparison, this is much less breaking than either of those changes.

**Comment by bcakmakoglu** [comment]
I'll probably add it earlier since 2.0.0 will take a while do finish.

**Comment by bcakmakoglu** [comment]
I'll put this into 1.29.0 probably ^^

**Comment by bcakmakoglu** [comment]
Fixed in `1.29.0`
